hedera-node/hedera-mono-service/src/test/java/com/hedera/services/store/contracts/StaticEntityAccessTest.java

```diff
@@ -295,6 +337,35 @@ void getsZeroBalanceIfNoAssociation() {
         assertEquals(0, subject.balanceOf(accountId, tokenId));
     }
 
+    @Test
+    void getKeys() {
+        token.setAdminKey(key);
```

Comment:
This doesn't really ensure that the correct key is returned so much as a key is returned.
For complete coverage it'd be better to make the keys unique for confirmation

Reply:
DONE